.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{md,mdx}]
```

Comment:
I would treat all files the same for the sake of simplicity. no max length will make it incredibly hard to parse long text. I'd be totally in on enforcing that

Reply:
That can really break markdown files. Enabling line wrap should make it easy to follow the text.

Reply:
Markdown syntax relies a lot on line break. I would suggest we keep this option as it is